.golangci.yml

```diff
@@ -43,11 +42,7 @@ linters-settings:
 
   gocyclo:
     # minimal code complexity to report, 30 by default (but we recommend 10-20)
-    min-complexity: 10
-
-  maligned:
-    # print struct with more effective memory layout or not, false by default
-    suggest-new: true
+    min-complexity: 20
```

Comment:
Was this supposed t change from 10 to 20?

Reply:
good catch, this was specific for function-azresourcegraph, will amend